core/go.mod

```diff
@@ -41,6 +41,10 @@ require (
 )
 
 require (
+	cloud.google.com/go v0.112.1 // indirect
+	cloud.google.com/go/compute/metadata v0.3.0 // indirect
+	cloud.google.com/go/iam v1.1.6 // indirect
+	cloud.google.com/go/storage v1.39.1 // indirect
```

Comment:
I appreciate that we've tried to keep this neutral, and that we think it will support other cloud storage providers as well, but I'm a bit dubious about these google-cloud specific dependencies.

Can they be avoided?

Will users be happy to see these?

Will they significantly increase the binary size?

cc @beeme1mr @thisthat @Kavindu-Dodan

Reply:
I think I understand a bit better - these are just the drivers that power the gocloud stuff, right? I guess as long as our API service is still neutral, I'm happy... as long as the bin isn't suddenly massive.

Reply:
These deps cannot be avoided as we have a direct dependency on gcs.